### ISSUE TITLE
Consider retiring ddev-proxy-support - should be obsolete

### ISSUE THREAD

**Issue by rfay** [drud/ddev-proxy-support#5]
It looks to me like 
* https://github.com/docker/compose/issues/9034

being solved may make this add-on obsolete, just as you signed on to maintainer @DaRealFreak ! My quick testing says that docker-compose now works with proxies without these aids. 

Would appreciate if you could test using latest DDEV HEAD and see if you agree. `brew install --HEAD --fetch-head ddev` or grab binaries from https://github.com/drud/ddev/actions/runs/3355723146

See also 
* https://github.com/drud/ddev/pull/4350

**Comment by DaRealFreak** [comment]
yes I tested it just now and can confirm that it also works for me

**Comment by rfay** [comment]
And another fix:
* https://github.com/docker/compose/pull/10159

**Comment by rfay** [comment]
I don't think we need this any more. If anybody is using, try not using it and see what happens.

**Comment by binary-data** [comment]
@rfay Please keep this one, if it doesn't require much maintenance!

I've tried using the Docker proxy configuration as described here: https://docs.docker.com/engine/cli/proxy/#configure-the-docker-client

However, this broke a couple of DDEV features, specifically:

- DNS/Router. I was only able to access my app via https://127.0.0.1:32227 link
- XDebug. I couldn't use it at all

So I resorted to using ddev-proxy-support and it worked! I could use DDEV as usual, and proxy settings worked. It was great to have a fallback option instead of spending more hours digging through Docker proxy settings.

**Comment by rfay** [comment]
Thanks @binary-data - I would have thought you could just used these env vars and everything would work, don't think you should have to make changes to docker proxy config.

```
HTTP_PROXY=yours
HTTPS_PROXY=yours
NO_PROXY=no-proxy-addrs
http_proxy=${HTTP_PROXY}
```

The docker CLI is not used here...

**Comment by rfay** [comment]
Related:
* https://github.com/ddev/ddev/issues/7044
